backend/src/controllers/auth.ts

```diff
@@ -29,11 +29,12 @@ const reqLogin = async (err: Error, user: User, req: Request, res: Response, nex
 }
 
 export const signUp = async (req: Request, res: Response, next: NextFunction) => {
-  const doesUserExit = await UserModel.exists({ email: req.body.email });
-  if (doesUserExit) {
+  const doesUserExist = await UserModel.exists({ email: req.body.email });
```

Comment:
just check if they are already a counselor here, merit award wants us to allow multiple user types at the same time so its chill